migrations/2_deploy.js

```diff
@@ -55,30 +53,73 @@ const TokenA = artifacts.require("TokenA");
 const TokenB = artifacts.require("TokenB");
 const FeeToken = artifacts.require("TokenC");
 
+// External contracts
+let { ZeroExExchangeV1, ZeroExProxyV1 } = require("../test/contracts/ZeroExV1");
+let { ZeroExExchangeV2, ZeroExProxyV2 } = require("../test/contracts/ZeroExV2");
+
 // Other constants
 const BigNumber = require('bignumber.js');
 const ONE_HOUR = new BigNumber(60 * 60);
 
-function maybeDeployTestTokens(deployer, network) {
+// Helper functions
+function parseExternalContracts(contracts) {
+  const classDefaults = {
+    from: web3.eth.accounts[0],
+    gas: 6721975,
+    gasPrice: 100000000000
+  };
+  for (let i in contracts) {
```

Comment:
does this work? won't this give you the items themselves and not the indexes?

Reply:
javascript gives the indexes